cmd/collector/main.go

```diff
@@ -225,10 +226,13 @@ func startZipkinHTTPAPI(
 		r := mux.NewRouter()
 		zHandler.RegisterRoutes(r)
 
+		origins := strings.Split(strings.Replace(allowedOrigins, " ", "", -1), ",")
+		headers := strings.Split(strings.Replace(allowedHeaders, " ", "", -1), ",")
```

Comment:
we shouldn't do this in main, it is better to encapsulate this into the builder, similar to this:

https://github.com/jaegertracing/jaeger/blob/4ed618d934a132407b62888969147557b1fcb796/cmd/agent/app/reporter/grpc/flags.go#L47-L50